share/spice/quandl/commodities/quandl_commodities.js

```diff
@@ -0,0 +1,137 @@
+/**
+* Created with DuckDuckHack.
+* User: brianrisk
+* Date: 2015-07-10
+* Time: 12:00 PM
+* To change this template use Tools | Templates.
+*/
+
+
+(function (env) {
+    'use strict';
+    env.ddg_spice_quandl_commodities = function(api_result){
+
+        if (!api_result) {
+            return Spice.failed('quandl_commodities');
+        }
+
+        var result = api_result;
+
+        // we need two data points to get percent change
+        if (result.data.length < 2) {
+            return Spice.failed('quandl_commodities');
+        }
+        
+        var quandlCode = result.source_code + "/" + result.code;
+        
+        // setting title and units based on database/dataset string
+        result.header = "Price of ";
+        if (quandlCode === "OFDP/ALUMINIUM_21") {
```

Comment:
May be a little cleaner to use a `switch` here, or better yet, an object that maps the codes to the header and unit so you have:

```js
var commodities = {
    "OFDP/ALUMINIUM_21": {
        header: "Aluminium",
        units: "$USD/tonne",
    },
    ...
}

result.header = "Price of " + commodities[quandleCode].header;
result.units = commodities[quandleCode].units;
```